ruma-events-macros/src/content_enum.rs

```diff
@@ -2,10 +2,29 @@
 
 use proc_macro2::TokenStream;
```

Comment:
All of the changes in this module seem to just be shuffling things around, can we remove those?

Reply:
Can I just remove `ruma-events-macros/src/content_enum.rs` the `event_enum.rs` file generates the content enums already.

Reply:
Sure, if `event_enum` does everything, please remove `content_enum`.